ann_benchmarks/algorithms/pgvector/module.py

```diff
@@ -62,3 +62,55 @@ def get_memory_usage(self):
 
     def __str__(self):
         return f"PGVector(m={self._m}, ef_construction={self._ef_construction}, ef_search={self._ef_search})"
+
+
+class PGVectorIVF(BaseANN):
```

Comment:
If this is to be added, I'd recommend using the original implementation, though adding the binary load found in https://github.com/erikbern/ann-benchmarks/pull/488:

https://github.com/erikbern/ann-benchmarks/tree/a884247a83db918d4f19c82048293c8fce65b30b/ann_benchmarks/algorithms/pgvector

Reply:
I wasn't aware it was part of the ann_benchmark workload. It makes sense to bring back the original implementation, but I don't know why it was removed in the first place; I am trying to learn the tradeoffs presented by both algorithm implementations and find it extremely useful.